lib/fx/schema_dumper/function.rb

```diff
@@ -23,9 +23,11 @@ def empty_line(stream)
       end
 
       def functions(stream)
+        stream.puts('  execute "SET check_function_bodies=off"')
```

Comment:
If we are going down this route, this should be performed on the adapter level to have a `{pre,post}_function_dump`.

I have yet to think about this, or seen it. So there is some reading up for me to do.